app/templates/route-components/buttons.hbs

```diff
@@ -152,3 +165,60 @@ html[lang^='zh'] &__start-a-wiki-button {
 		{{#wds-button tagName='a' href='#' class='wds-is-secondary'}}Secondary link button{{/wds-button}}
 	{{/component-demo}}
 </section>
+
+<section>
+	<h2 id="group">Button group</h2>
+	<p>Group a series of buttons together on a single line with the button group.</p>
+	<p>Button group comes with no default styling of it's own. Colors, sizes and animations are driven by buttons.</p>
+	<p>Wrap a series of buttons with <code>.wds-button</code> in <code>.wds-button-group</code>.</p>
+	{{#component-demo}}
+		<div class="wds-button-group">
+			{{#wds-button}}1{{/wds-button}}
+			{{#wds-button}}2{{/wds-button}}
+			{{#wds-button disabled=true}}3{{/wds-button}}
+			{{#wds-button}}4{{/wds-button}}
+			{{#wds-button}}5{{/wds-button}}
+		</div>
+	{{/component-demo}}
+	<p>Mixed type buttons are not supported. All buttons have to be of the same type.</p>
```

Comment:
by mixed type buttons you mean primary/secondary, but actually buttons with/without icon (and with/without dropdown in the future) are supported I think